package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "REACT_APP_VERSION=$npm_package_version FAST_REFRESH=false react-scripts start",
     "build": "REACT_APP_VERSION=$npm_package_version react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!axios)/\"",
```

Comment:
Hi @DnsZhou , 

can you check this change it ok from your point of view? https://github.com/axios/axios/issues/5026#issuecomment-1271194848

Reply:
Hi @matedo1 ,as discussed in the axios issue,  after 1.0.0+ it refactored as an ES module, instead of common js module, which would cause issue with jest. I am fine with the workaround and I don't see any blocker as far as what we are working on, and I verified this solution with a clean build from my local env.